documentation/software_examples/other_languages_tools/conda-container.md

```diff
@@ -0,0 +1,178 @@
+- **Miniconda** is a slim Python distribution, containing the minimum amount of packages necessary for a Python installation that can use conda.
+- **Anaconda** is a pre-built scientific Python distribution based on Miniconda that has many useful scientific packages pre-installed.
+
+<p style="color:blue;">To create the smallest, most portable Python installation possible, we recommend starting with Miniconda and installing only the packages you actually require.</p>
+
+To use a Miniconda installation for your jobs, create an Apptainer/Singularity definition file and
+build it (general instructions [here](../../../htc_workloads/using_software/containers-singularity/)).
+
+## Apptainer/Singularity Definition File
+
+The definition file tells Apptainer/Singularity how the container should be built,
+and what the environment setup should take place when the container is instantiated.
+In the following example, the container is based on Ubuntu 22.04. A few base
+operating system tools are installed, then Miniconda, followed by a set of 
+`conda` commands to define the Conda environment. The `%environment` is used
+to ensure jobs are getting the environment activated before the job runs. To build
+your own custom image, start by modifing the `conda install` line to include
+the packages you need.
+
+
+    Bootstrap: docker
+    From: ubuntu:22.04
+    
+    %environment
+        # set up environment for when using the container
+        . /opt/conda/etc/profile.d/conda.sh
+        conda activate
+    
+    %post
+        # base os
+        apt-get update -y
+        apt-get install -y build-essential wget
+    
+        # install miniconda
+        wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+        bash Miniconda3-latest-Linux-x86_64.sh -b -f -p /opt/conda
+        rm Miniconda3-latest-Linux-x86_64.sh
+    
+        # install conda components - add the packages you need here
+        . /opt/conda/etc/profile.d/conda.sh
+        conda activate
+        conda install -y -c conda-forge numpy cowpy
+        conda update --all 
+
+
+The next step is to build the image. Run:
+
+    $ apptainer build my-container.sif image.def
+
+You can explore the container locally to make sure it works as expected with the shell subcommand:
+
+    $ apptainer shell my-container.sif
+
+This example will give you an interactive shell. You can explore the
+container and test your code with your own inputs from your `/home`
+directory, which is automatically mounted (but note - $HOME will not be
+available to your jobs later). Once you are down exploring, exit the
+container by running `exit` or with `CTRL+D`
+
+It is important to use the correct transfer mechanism to get the 
+image to your job. Please make sure you use [OSDF](../../../htc_workloads/managing_data/osdf/)
+and version your container in the filename. For example:
+
+    $ cp my-container.sif /ospool/protected/<username>/my-container-v1.sif
```

Comment:
Maybe replace /protected with /public or something that will be around later this year?